config/yurtctl-servant/setup_edgenode

```diff
@@ -114,16 +103,15 @@ preset() {
 # setup_yurthub sets up the yurthub pod and wait for the its status to be Running
 setup_yurthub() {
     provider=$1
-    # 1. put yurt-hub yaml into /etc/kubernetes/manifests 
-    if [ "$provider" == "minikube" ]; then
-        log "setting up yurthub on nodes of minikube"
-        yurthub_yaml=$(echo "$YURTHUB_TEMPLATE" | 
-            sed 's|__pki_path__|/var/lib/minikube/certs|')
-    else
-        log "setting up yurthub on nodes of ack"
-        yurthub_yaml=$(echo "$YURTHUB_TEMPLATE" | 
-            sed 's|__pki_path__|/etc/kubernetes/pki|')
-    fi 
+    # 1. put yurt-hub yaml into /etc/kubernetes/manifests
+    log "setting up yurthub on nodes"
+    KUBELET_CONF=`ps -aux | grep kubelet | grep '\-\-kubeconfig' | tr ' ' '\n' | grep '\-\-kubeconfig' | awk -F '='  '{print $2}'`
+    apiserver_addr=`cat $KUBELET_CONF | grep "server:" | awk  '{print $2}'`
```

Comment:
how about add a snippet to verify the apiserver_addr is valid or not? if invalid, we need log error and exit revert.

Reply:
add